metcalcpy/util/vcnt_statistics.py

```diff
@@ -512,3 +511,52 @@ def calculate_vcnt_dir_abser(input_data, columns_names, aggregation=False):
         result = None
     warnings.filterwarnings('ignore')
     return result
+
+
+def calculate_vcnt_anom_corr(input_data, columns_names, aggregation=False):
+    warnings.filterwarnings('error')
+    try:
+        total = get_total_values(input_data, columns_names, aggregation)
+        n = len(input_data)# n
```

Comment:
I *think* that this is the issue.
```suggestion
        n = total
```
Looking at this line of code in MET:
https://github.com/dtcenter/MET/blob/d91b103d8b0a613b0773fdace67e84ae8d0ed3ba/src/libcode/vx_statistics/met_stats.cc#L1799
```
      n = vacount;

      // Check for bad data since new are not present for previous versions
      if(is_bad_data(fa_speed_bar) || is_bad_data(oa_speed_bar)) {
         ANOM_CORR.v = bad_data_double;
      }
      else {
         ANOM_CORR.v = compute_corr(fa_speed_bar*n, oa_speed_bar*n,
                                    uvffa_bar*n, uvooa_bar*n,
                                    uvfoa_bar*n, n);
      }
```
The value of `n` is the number of pairs being aggregated... so basically the sum of the `TOTAL` column. So we want it to be 29040+29040 and not 2. Maybe that'll give us the desired result?

Reply:
@JohnHalleyGotway  I fixed the anom_corr computation. It should produce the expected result